.github/workflows/ci.yml

```diff
@@ -234,17 +234,3 @@ jobs:
               - 'packages/integrations/*/README.md'
               - 'packages/astro/src/@types/astro.ts'
               - 'packages/astro/src/core/errors/errors-data.ts'
-
-      - name: Build autogenerated docs pages from current astro branch
-        if: ${{ steps.changes.outputs.docs == 'true' }}
-        run: cd smoke/docs && pnpm docgen && pnpm docgen:errors && pnpm docgen:integrations
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SOURCE_REPO: ${{ github.event.pull_request.head.repo.full_name || github.event.repository.full_name }}
-          SOURCE_BRANCH: ${{ github.head_ref || github.ref_name }}
-
-      - name: Test
-        run: pnpm run test:smoke
-        env:
-          SKIP_OG: true
-          PUBLIC_TWO_LANG: true
```

Comment:
We don't need to remove the entire smoke test. Just the `docgen:integrations` generation. I can't inline suggest, but here's what it should probably look like now

```
      - name: Build autogenerated docs pages from current astro branch
        if: ${{ steps.changes.outputs.docs == 'true' }}
        run: cd smoke/docs && pnpm docgen && pnpm docgen:errors
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          SOURCE_REPO: ${{ github.event.pull_request.head.repo.full_name || github.event.repository.full_name }}
          SOURCE_BRANCH: ${{ github.head_ref || github.ref_name }}

      - name: Test
        run: pnpm run test:smoke
        env:
          SKIP_OG: true
          PUBLIC_TWO_LANG: true
```

Reply:
🥳 passing again